### ISSUE TITLE
Invoke-EnumerateLocalAdmin Error

### ISSUE THREAD

**Issue by cynicXer** [PowerShellMafia/PowerSploit#126]
This error is generated when calling Invoke-EnumerateLocalAdmin with no command line arguments.  The command executes successfully and gives me a list of local administrators, but I still get the following error:

`<Objs Version="1.1.0.1" xmlns="http://schemas.microsoft.com/powershell/2004/04"><S S="warning">[!] Error: Exception calling "Invoke" with "2" argument(s): "The network address is invalid._x000D__x000A_"</S><S S="warning">[!] Error: Exception calling "Invoke" with "2" argument(s): "The network address is invalid._x000D__x000A_"</S><S S="warning">[!] Error: Exception calling "Invoke" with "2" argument(s): "The network address is invalid._x000D__x000A_"</S></Objs>`

When calling with additional arguments (in this case I was using -computeradspath to restrict the tested group and -outfile) I got the following error.  No other output is generated.

`<Objs Version="1.1.0.1" xmlns="http://schemas.microsoft.com/powershell/2004/04"><S S="warning">[!] Error: Exception calling "Invoke" with "2" argument(s): "Unknown error (0x80005000)"</S></Objs>`

It seems that providing any parameters to the script causes it to fail with the above error.

Testing was conducted in my test Active Directory environment from an unprivileged domain user account.

It should be noted that I got the same error from the Master and Dev branch code.

**Comment by cynicXer** [comment]
It should be noted that I got the same error from both the Master and Dev branch code.

**Comment by HarmJ0y** [comment]
So those blurbs are not actually errors, just dressed up output from a Write-Warning (hence <S S="warning">) that actually resides in Get-NetLocalGroup.

Try changing this line https://github.com/PowerShellMafia/PowerSploit/blob/dev/Recon/PowerView.ps1#L7312 to "Write-Verbose" instead, and let me know if that fixes it (if so I'll patch it into the dev branch).

**Comment by HarmJ0y** [comment]
@cynicXer - any luck? We'll close this issue out in a few weeks if there's no response

**Comment by cynicXer** [comment]
Wow, I'm a lazy bum.  There must have been a change since I posted this issue, line 7312 doesn't have a Write-Verbose command in it.  I updated my local repo and changed the Write-Warning in Get-NetLocalGroup from Write-Warning to Write-Verbose in PowerView.ps1, and that appears to have fixed the first error.  It also looks like Invoke-EnumerateLocalAdmin doesn't accept -recurse as an argument, which would explain the second error.  

Thank you!
